src/EAction.h

```diff
@@ -41,14 +41,15 @@ class EAction : public QAction
 public:
     EAction(QIcon&, QString&);
 
-    int mID;
-    QPointer<Host> mpHost;
-
 public slots:
     void slot_execute(bool checked);
 
 signals:
     void triggered(QAction*);
+
+public://private:
```

Comment:
This doesn't. Making them private now requires more changes. This is effectively a TODO.